src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -972,11 +973,19 @@ class ContentSidebar extends PureComponent<Props, State> {
      * @return {void}
      */
     fetchCurrentUser(user?: User, shouldDestroy?: boolean = false): void {
+        const { fileId = '' } = this.props;
         if (shouldRenderSidebar(this.props)) {
             if (typeof user === 'undefined') {
+                const getUrl = this.api.getUsersAPI(false).getUrl();
                 this.api
                     .getUsersAPI(shouldDestroy)
-                    .get('', this.fetchCurrentUserSuccessCallback, this.fetchCurrentUserErrorCallback);
+                    .makeRequest(
```

Comment:
You should not bleed api stuff in here as much as possible.
Change the get function to to take an id and provide it instead.